src/hotspot/cpu/ppc/assembler_ppc.hpp

```diff
@@ -350,6 +350,7 @@ class Assembler : public AbstractAssembler {
 
     SETBC_OPCODE  = (31u << OPCODE_SHIFT | 384u << 1),
     SETNBC_OPCODE = (31u << OPCODE_SHIFT | 448u << 1),
+    SETBCR_OPCODE = (31u << OPCODE_SHIFT | 416u << 1),
```

Comment:
update copyright header.

Reply:
Adding IBM Corp or just he Oracle header ?

Reply:
Oh, I was just asking about updating copyright header years. Make it 2024. for both SAP and Oracle.